bin/launch.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+if [[ ! -f ".env" ]]; then
+    cp .env.example .env
+fi
+
+docker-compose up -d
+
+sleep 2
+python -m webbrowser http://$(grep SERVER_NAME .env | xargs | cut -d "=" -f 2)
```

Comment:
nice one, but is 2 going to be enough time?

Reply:
:o) this may throw a 502 on a new build but OK in local dev. We can adjust in future if necessary